ngx-fudis/projects/ngx-fudis/src/lib/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,4 +1,4 @@
-<nav [attr.aria-label]="breadcrumbsAriaLabel" *ngIf="links.length > 0">
+<nav [attr.aria-label]="_breadcrumbsPrefix + ': ' + breadcrumbsLabel" *ngIf="links.length > 0">
 	<ul class="fudis-breadcrumbs">
 		<li class="fudis-breadcrumbs__item" *ngFor="let link of links; let last = last">
 			<fudis-link
```

Comment:
Should we have 'aria-current' attribute set to the last link in the breadcrumbs?

Reply:
There is aria-current in fudis-link element and it works like this at the moment. Do we need something else?

<img width="867" alt="Screenshot 2023-10-26 at 9 22 43" src="https://github.com/funidata/fudis/assets/92301174/dc258b85-deb3-4052-b884-4cdb7ce014f3">